printing/ppvernac.ml

```diff
@@ -967,8 +967,13 @@ module Make
           )
         (* Solving *)
       | VernacSolve (i,info,tac,deftac) ->
+        let pr_range_selector (i, j) =
+          if i = j then int i
+          else int i ++ str"-" ++ int j
+        in
         let pr_goal_selector = function
           | SelectNth i -> int i ++ str":"
+          | SelectList l -> prlist_with_sep (fun () -> str ", ") pr_range_selector l
```

Comment:
It would, yes. I noticed there was a similar problem with the "[id]" and "par" selectors, too, I'll submit another PR for that.